contracts/koh/contracts/KingAutomaton.sol

```diff
@@ -191,34 +221,30 @@ contract KingAutomaton {
 
   // Pay for multiple slots at once, 32 seems to be a reasonable amount.
   function payForGas(uint256 ballot_box_id, uint256 _slotsToPay) public validBallotBoxID(ballot_box_id) {
-    BallotBox storage p = ballot_boxes[ballot_box_id];
-    require((slots.length - p.paidSlots) >= _slotsToPay, "Too many slots!");
-    uint _newLength = p.paidSlots + _slotsToPay;
+    BallotBox storage b = ballot_boxes[ballot_box_id];
+    require((slots.length - b.paidSlots) >= _slotsToPay, "Too many slots!");
+    uint _newLength = b.paidSlots + _slotsToPay;
```

Comment:
Also pre-fetch these two in memory to save on gas costs
```
  uint256 _votesWords = b.votesWords;
  uint256 _slotsLength = slots.length;
```

Reply:
First one is unnecessary